test/fixtures/definitions.txt

```diff
@@ -1,47 +1,53 @@
-Indented text blocks are rendered as blockquotes
+Indented text blocks (blockquotes) are rendered as paragraphs with a class of "indent-*"
 .
 Paragraph text
 
     This is a blockquote.
     It can span multiple lines.
 .
 <p>Paragraph text</p>
-<blockquote>This is a blockquote.
-It can span multiple lines.</blockquote>
+<p class="indent-1">This is a blockquote.
+It can span multiple lines.</p>
 .
 
-Blockquotes are rendered without paragraph tags
+Indented text blocks are separated by blank lines
 .
 Paragraph text
 
-    This is the first paragraph of a blockquote.
+    Blockquote text
 
-    This is the second paragraph of a blockquote.
+    Blockquote text
+    
```

Comment:
There are four spaces here, so this will be rendered to 
<p class="indent-1">Blockquote text

Blockquote text</p>